src/sys/windows.rs

```diff
@@ -1022,36 +1035,41 @@ impl FromRawSocket for crate::Socket {
     }
 }
 
-#[test]
```

Comment:
Why did you make these changes? Please revert.

Reply:
if you don't warp #[test] in cfg(test), then these code will be compiled into the binary.